src/app/pages/receive/receive-collectible/receive-collectible-oridinal.tsx

```diff
@@ -32,27 +33,51 @@ export function ReceiveCollectibleOrdinal() {
           <Title fontSize="20px" lineHeight="28px" px="extra-loose">
             Deposit Ordinal inscription
           </Title>
-          <ErrorLabel>Important to keep in mind</ErrorLabel>
-          <Text color={color('text-title')} fontSize={1} px={['base', '40px']}>
-            This Taproot address is for Ordinals only. Do not use this address more than once for
-            depositing an inscription
+          <Text color={color('text-title')} fontSize={1} px={['50px', '70px']}>
+            Use your wallet to create inscriptions within minutes in
```

Comment:
> Use [the] wallet to create insciptions [...] *in*?

Maybe

> Use [the] wallet to create insciptions [...] *with*?

Reply:
@mica000 or @markmhx what do you think here? I also paused on this ...could use `here` too? I do think `in` seems off?

Reply:
Wouldn't it be `on gamma.io`?

Reply:
...with these apps: